### ISSUE TITLE
Investigation Clue Context

### ISSUE THREAD

**Issue by Nynrose** [Arx-Game/arxcode#100]
**Is your feature request related to a problem? Please describe.**

When you receive an inform about investigations, you get a snippet in the form saying something like:

While looking into Secret #1 of Evelyn, you found some references to 'Godtouched' that resulted in your discovery.

I had thought that also appended to the clue as a note for context of why you had the clue, but it doesn't actually do that, and I think it might be a great aid if it appends the same information as the @inform. Instead, you just get:

Your investigation has discovered this!

**Describe the solution you'd like**

Consider appending the snippet of context you get in the @inform to the clues that a person is researching so they can remember what they were researching if they look at the clue, instead of just: Your investigation has discovered this!

I think this option will be a big help moving forward, especially now that people need to use tags to investigate things.

**Comment by TehomCD** [comment]
Yeah, it'd be a good idea to add that into the inform for additional context.

**Comment by Nynrose** [comment]
The @inform already does it. What I'm proposing is to include it in the @clue note that's attached to clues when you discover the research material, and not just: Your investigation discovered this! - which is the default information you see when you look at clues you've discovered with the @clue command. This would be a going forward thing, obviously :)

**Comment by TehomCD** [comment]
Ah, okay. Yeah, that's the appended ClueDiscovery message.

**Comment by Selerik** [comment]
Adding on to this, it would be nice if it told you in
the @investigate/@helpinvestigate what clue the investigation resulted in,
if the investigation concluded. A two-way bridge so the information links
from both sides.
